app/posts/register-trainee-teachers/2024-07-30-using-hesa-to-submit-trainee-records-for-2024-to-2025.md

```diff
@@ -0,0 +1,77 @@
+
+Accredited providers are responsible for submitting all trainee records. This includes anyone who is directly trained by a lead provider.
+
+Because lead partners are not responsible for trainee records, they can view their trainee records but do not have permission to create or edit trainee records in Register.
+
+## Supporting accredited providers and lead partners to submit trainee data
+
+In most cases where there is an accredited provider and a lead partner, ITT trainees will be located with their lead partner. We expect lead partners to use their student record system to track trainees, just like they did before losing ITT accreditation.
+
+Lead partners are still training providers, mostly universities, responsible for tens of thousands of students, with ITT trainees making up only a small percentage of their student population.
+
+To follow the strictest interpretation of the responsibilities between an accredited provider and a lead partner, all records should be created and submitted by the accredited provider.
+
+We considered if it was possible to help accredited providers submit records for all their trainees, including ones studying with lead partners. The two possible options were either making changes to how HESA worked or creating a scenario where it was likely that lead partners would need to transfer trainee records to their accredited provider.
+
+With a year left of HESA performing the ITT collection and a relatively short time to make the change, it was not practical to make changes to how HESA operates.
+
+Asking lead partners to send their data to accredited providers and then submit it to Register is likely to create a large administrative burden for both parties and invites an unacceptable level of risk.
+
+We decided it was necessary to let lead partners submit ITT trainee data through HESA as they are used to. Register will then associate the trainee records with the corresponding accredited provider.
+
+### Associating trainee records to the correct accredited provider
+
+We explored with HESA using the `SDLEAD` field for lead partners to indicate their accredited provider by reference to that organisation’s unique reference number (URN). However, this would require significant manual intervention by the lead partner before submitting to HESA, which seemed burdensome and potentially error prone.
+
+Instead, we will attribute records to the correct accredited provider after receiving them.
+
+We will decide the best way to do this accurately. Publish teacher training courses (Publish) has organisation types that Register can reference, however:
+
+- it will take significant work to automate
+
+- the organisation types are not identical as Register (‘Training partners’ instead of ‘Lead partners’)
```

Comment:
@ralph-hawkins 'Training partner' is a synonym of 'Lead partner', so wouldn't prevent synchronisation between Publish and Register. 

'Training partner' is used in the user interface (specifically the primary navigation) rather than data.

cc. @defong